integration-test/PythonSpec.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE TupleSections #-}
+
```

Comment:
Could you remove this newline?

Reply:
Interesting! This looks like a minor bug in hlint